src/clj/rems/ext/mondo.clj

```diff
@@ -91,7 +92,9 @@
            compressed-format
            vec))))
 
-(def ^:private supported-mondo-release-tag "the version of Mondo we support so far" "v2021-10-01")
+(def ^:private supported-mondo-release-tag
+  "the version of Mondo we support so far"
+  "v2022-08-01")
```

Comment:
Are you using some new formatting settings with a smaller column limit?

Reply:
as discussed, this was my hand-editing. i've been trying to limit my code to max 100 characters so i can work more efficiently on laptop when working abroad (my eyesight isn't getting any better so i need to crank up editor font size)

Reply:
Ok I have a soft limit of around 120. And I do crank up the editor font size to MAX all the time :)